tests/test_data/felix.kitchen/felixkitchen.json

```diff
@@ -0,0 +1,66 @@
+{
+  "author": "FEL!X KITCHEN",
+  "canonical_url": "https://felix.kitchen/2024/04/07/piccata-milanese-schnitzel-parmesan-safran-risotto-petersilie/",
+  "host": "felix.kitchen",
+  "description": "Ob das Schnitzel nach Mailänder-Art original italienisch ist, oder ob die Panade nördlich der Alpen mit Parmesan sozusagen «italienisiert» wurde, bleibt dahingestellt.\nTatsache ist: die so umhüllten und gebratenen Schnitzelchen sind ganz einfach lecker! Und was könnte dazu besser passen, als ein goldgelber Safran-Risotto?!",
+  "image": "https://felix.kitchen/wp-content/uploads/2023/09/piccata_milanese_beitragsbild.jpg",
+  "ingredients": [
+    "15 g Parmesan am Stück",
+    "1/2 unbehandelte Zitrone",
+    "1 Ei (kl.)",
+    "1 EL Wasser",
+    "1–2 EL Weizen­mehl",
+    "1 TL Senf, mittel­scharf",
+    "Salz, Pfeffer aus der Mühle",
+    "2 Kalbs- oder Schweine­schnitzel à ca. 125 g",
+    "Salz, Pfeffer aus der Mühle",
+    "Weizen- oder Reismehl",
+    "1 EL Olivenöl",
+    "1 EL Butter",
+    "etwas glatte Peter­­silie"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "15 g Parmesan am Stück",
+        "1/2 unbehandelte Zitrone",
+        "1 Ei (kl.)",
+        "1 EL Wasser",
+        "1–2 EL Weizen­mehl",
+        "1 TL Senf, mittel­scharf",
+        "Salz, Pfeffer aus der Mühle",
+        "2 Kalbs- oder Schweine­schnitzel à ca. 125 g",
+        "Salz, Pfeffer aus der Mühle",
+        "Weizen- oder Reismehl",
+        "1 EL Olivenöl",
+        "1 EL Butter",
+        "etwas glatte Peter­­silie"
+      ],
+      "purpose": null
+    }
+  ],
+  "instructions": "1PICCATA-TEIG:\nParmesan sehr fein reiben (Micro-Zestenreibe!), ebenfalls die Schale der halben Zitrone.\nMit den weiteren Zutaten mischen, kurz ruhen lassen.\nDer Teig sollte nicht allzu dick sein.\n2Fleisch nach Belieben in kleinere Stücke schneiden, zwischen Klarsichtfolie leicht plattieren, es sollte nicht zu dünn werden (Fleischklopfer oder Pfannenboden).\nSaft der Zitronenhälfte auspressen, das Fleisch darin kurz marinieren.\nKann bis hierher vorbereitet werden!\n3Fleisch zwischen Küchen­papier trocknen, würzen.\nIm Mehl wenden, dann durch den Teig ziehen, was am ein­fachsten mit der Koch­pinzette geschieht.\n4Portionenweise in der Öl-Butter-Mischung beidseitig ausbacken.\nAuf Küchenpapier abtropfen lassen.\nIm Ofen bei 80 °C (O+U) warm stellen.\n5Währenddessen einen Safran-Risotto zubereiten, siehe hier.\n6Risotto zusammen mit der Piccata anrichten.\nMit gehackter Petersilie bestreuen.\nnicht scharf",
+  "instructions_list": [
+    "1PICCATA-TEIG:",
```

Comment:
Let's try to add a separator (perhaps simply a space character) between the numbers and the section titles in the instructions?  It might be a little tricky (detecting some of the HTML markup tags), but I think it'd be worthwhile to improve the output text.